clustered/client/src/test/java/org/ehcache/clustered/client/docs/GettingStarted.java

```diff
@@ -49,34 +49,33 @@ public void resetPassthroughServer() throws Exception {
 
   @Test
   public void clusteredCacheManagerExample() throws Exception {
-    // tag::clusteredCacheManagerExample
+    // tag::clusteredCacheManagerExample[]
     final CacheManagerBuilder<PersistentCacheManager> clusteredCacheManagerBuilder =
-        CacheManagerBuilder.newCacheManagerBuilder()
-            .with(ClusteringServiceConfigurationBuilder.cluster(URI.create("http://example.com:9540/my-application?auto-create")))
-            .withCache("simple-cache", CacheConfigurationBuilder.newCacheConfigurationBuilder(Long.class, String.class, heap(10)));
-    final PersistentCacheManager cacheManager = clusteredCacheManagerBuilder.build(true);
+        CacheManagerBuilder.newCacheManagerBuilder() // <1>
+            .with(ClusteringServiceConfigurationBuilder.cluster(URI.create("http://example.com:9510/my-application?auto-create"))); // <2>
```

Comment:
You updated the port which is good - can't we have the hostname be localhost as well?